components/_lists.scss

```diff
@@ -0,0 +1,19 @@
+.wds-list {
+	display: block;
+	list-style-type: none;
+	margin: 18px;
+	padding: 0;
+}
+
+.wds-list-item {
+	display: flex;
+	flex-direction: column;
+	font-size: $typescale-size-minus-1;
+	justify-content: center;
+	min-height: 36px;
```

Comment:
Should we use `min-height` or `padding` here? What would happen if there is a multiline item?